vhdbuilder/packer/cleanup.sh

```diff
@@ -28,15 +28,18 @@ fi
 
 #clean up managed image
 if [[ -n "$AZURE_RESOURCE_GROUP_NAME" && -n "$IMAGE_NAME" ]]; then
-  if [[ "$MODE" != "default" ]]; then
+  if [[ "$MODE" != "default" && ${ARCHITECTURE,,} != "arm64" ]]; then
     id=$(az image show -n ${IMAGE_NAME} -g ${AZURE_RESOURCE_GROUP_NAME} | jq .id)
     if [ -n "$id" ]; then
       echo "deleting managed image ${IMAGE_NAME} under resource group ${AZURE_RESOURCE_GROUP_NAME}"
       az image delete -n ${IMAGE_NAME} -g ${AZURE_RESOURCE_GROUP_NAME}
     fi
+  else
+    echo "Not attempting managed image deletion due to ARM64 architecture."
```

Comment:
what should we do with he ARM64 images ?  are they staying present ?

Reply:
No managed images are created for ARM64 SKUs, only image versions. That was leading to the error when we tried to delete them.

Reply:
are we sure about this? if so good catch

Reply:
yep!